docs/user/04-metrics.md

```diff
@@ -1,3 +1,395 @@
+
+```yaml
+kind: Secret
+apiVersion: v1
+metadata:
+  name: backend
+  namespace: default
+stringData:
+  endpoint: https://backend.example.com:4317
+  user: myUser
+  password: XXX
+  token: Bearer YYY
+```
+
+### Step 5: Rotate the Secret
+
+Telemetry Manager continuously watches the Secret referenced with the **secretKeyRef** construct. You can update the Secret’s values, and Telemetry Manager detects the changes and applies the new Secret to the setup.
+If you use a Secret owned by the [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator), you can configure an automated rotation using a `credentialsRotationPolicy` with a specific `rotationFrequency` and don’t have to intervene manually.
+
+### Step 6: Activate Prometheus-based metrics
+
+To enable collection of prometheus based metrics define a MetricPipeline having prometheus enabled as input:
+```yaml
+apiVersion: telemetry.kyma-project.io/v1alpha1
+kind: MetricPipeline
+metadata:
+  name: backend
+spec:
+  input:
+    application:
+      prometheus:
+        enabled: true
+  output:
+    otlp:
+      endpoint:
+        value: https://backend.example.com:4317
+```
+
+This approach assumes that you instrumented your application using a library like the [prometheus client library](https://prometheus.io/docs/instrumenting/clientlibs/), having a port in your workload exposed serving a typical Prometheus metrics endpoint.
+
+The agent is configured with a generic scrape configuration, which uses annotations to specify the endpoints to scrape in the cluster. 
+Having the annotations in place is everything you need for metrics ingestion to start automatically.
+
+Put the following annotations either to a service that resolves your metrics port, or directly to the pod:
+
+```yaml
+prometheus.io/scrape: "true"   # mandatory to enable automatic scraping
+prometheus.io/scheme: https    # optional, default is "http" if no Istio sidecar is used. When using a sidecar (Pod has label `security.istio.io/tlsMode=istio`), the default is "https". Use "https" to scrape workloads using Istio client certificates.
+prometheus.io/port: "1234"     # optional, configure the port under which the metrics are exposed
+prometheus.io/path: /myMetrics # optional, configure the path under which the metrics are exposed
+```
+
+> **NOTE:** The agent can scrape endpoints even if the workload uses Istio and accepts only mTLS communication.
+
+### Step 7: Activate runtime metrics
+To enable collection of runtime metrics for your pods, define a MetricPipeline having runtime enabled as input:
+```yaml
+apiVersion: telemetry.kyma-project.io/v1alpha1
+kind: MetricPipeline
+metadata:
+name: backend
+spec:
+  input:
+    application:
+      runtime:
+        enabled: true
+  output:
+    otlp:
+      endpoint:
+        value: https://backend.example.com:4317
+```
+
+### Step 8: Activate Istio metrics
+To enable collection of istio metrics for your pods, define a MetricPipeline having istio enabled as input:
```

Comment:
```suggestion
To enable collection of Istio metrics for your Pods, define a MetricPipeline with Istio enabled as input:
```

Is step 8 additional to step 7, or are they alternatives to each other?

Reply:
can be mixed as well, they are not exclusive to each other